modules/feed/html/rollup.js

```diff
@@ -291,44 +293,46 @@ exports.create = function (api) {
     var items = map(Array.from(ids), api.about.obs.name)
     return computed([items], (names) => names.map((n) => `- ${n}`).join('\n'))
   }
-}
 
-function twoDaysAgo () {
-  return Date.now() - (2 * 24 * 60 * 60 * 1000)
-}
 
-function many (ids, fn) {
-  ids = Array.from(ids)
-  var featuredIds = ids.slice(-4).reverse()
-
-  if (ids.length) {
-    if (ids.length > 4) {
-      return [
-        fn(featuredIds[0]), ', ',
-        fn(featuredIds[1]), ', ',
-        fn(featuredIds[2]), ' and ',
-        ids.length - 3, ' others'
-      ]
-    } else if (ids.length === 4) {
-      return [
-        fn(featuredIds[0]), ', ',
-        fn(featuredIds[1]), ', ',
-        fn(featuredIds[2]), ' and ',
-        fn(featuredIds[3])
-      ]
-    } else if (ids.length === 3) {
-      return [
-        fn(featuredIds[0]), ', ',
-        fn(featuredIds[1]), ' and ',
-        fn(featuredIds[2])
-      ]
-    } else if (ids.length === 2) {
-      return [
-        fn(featuredIds[0]), ' and ',
-        fn(featuredIds[1])
-      ]
-    } else {
-      return fn(featuredIds[0])
+  function twoDaysAgo () {
+    return Date.now() - (2 * 24 * 60 * 60 * 1000)
+  }
+
+  function many (ids, fn) {
+    ids = Array.from(ids)
+    var featuredIds = ids.slice(-4).reverse()
+
+    if (ids.length) {
```

Comment:
hey @gmarcos87, did you know [`intl-messageformat`](https://www.npmjs.com/package/intl-messageformat) which i setup has syntax for plural messages? https://formatjs.io/guides/message-syntax/#plural-format